src/api/api.service.ts

```diff
@@ -422,4 +422,38 @@ export class ApiService {
       ); 
     }
   }
+
+   async getAllApiContributors(apiId: string){
+    try {
+      //Check if Api exists
+      const api = await this.apiRepo.findOne({where:{id:apiId}})
+      if(!api){
+        throw new NotFoundException(
+          ZaLaResponse.NotFoundRequest(
+            "Not Found Error",
+            "Api Not Found",
+            "404"
+          )
+        )
+      }
```

Comment:
If you added IdCheck guard, there won't be need to check if api exists here or any other place.